presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/TaskDistribution.java

```diff
@@ -44,7 +44,7 @@ public StageCpuDistribution(
             @JsonProperty("min") long min,
             @JsonProperty("max") long max,
             @JsonProperty("total") long total,
-            @JsonProperty("average") double average)
+            @JsonProperty("average") float average)
```

Comment:
Just curious, why reduce precision?

Reply:
For current purpose of using this class for either memory bytes or cpu time,  float precision is enough.

Reply:
Changed it back `double` to avoid confusion